crates/context/src/lib.rs

```diff
@@ -217,6 +218,23 @@ impl ContextManager {
                 )
             }
 
+            let context_id: IdentityContextId = context.id.rt().expect("infallible conversion");
+            let member_id = identity_secret
+                .public_key()
+                .rt()
+                .expect("infallible conversion");
+
+            let nonce: u64 = this
+                .config_client
+                .query::<ContextConfigEnv>(
+                    this.client_config.new.protocol.as_str().into(),
+                    this.client_config.new.network.as_str().into(),
+                    this.client_config.new.contract_id.as_str().into(),
+                )
+                .fetch_nonce(context_id, member_id)
+                .await?;
+            let nonce = nonce + 1;
```

Comment:
no need to add 1, the contract does that already

Reply:
Updated